### PR TITLE
Fix languages x2

### PR DESCRIPTION
**Pull request by ItzSkyReed**

### CODE REVIEW

File: text2mcbook.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import textwrap
 
 try:
-    f = open("text.txt", "r", encoding='utf-8')
```
Comment (ItzSkyReed) [inline text2mcbook.py:5]: You don't need "r" here because it is a default value, so I deleted it

**Comment by Neocky** [comment]
### Hey @ItzSkyReed,
thank you so much for your PR. :)
I am just a little bit confused about your PR. You made some good changes with the Minecraft `give` command  already as file output but you later removed it again. Currently you just added the `encoding` part in the code and removed the one `"r"`.
If this is correct then I can add your 2 lines as submission.
If you wanted to change more than feel free. 😃

File: text2mcbook.py (hunk 1 above)
Comment (Neocky) [reply text2mcbook.py:5]: LGTM

**Comment by ItzSkyReed** [comment]
I made this code for myself, it is raw, and it seemed to me that it was not worth adding it to pull request. I want to finalize it, and then I will create a new PR.

**Pull request merged**